types/json/binary.go

```diff
@@ -237,6 +274,13 @@ func (bj BinaryJSON) valEntryGet(valEntryOff int) BinaryJSON {
 		}
 		totalLen := uint32(lenLen) + uint32(strLen)
 		return BinaryJSON{TypeCode: tpCode, Value: bj.Value[valOff : valOff+totalLen]}
+	case TypeCodeOpaque:
+		strLen, lenLen := uint64(bj.Value[valOff+1]), 1
+		if strLen >= utf8.RuneSelf {
+			strLen, lenLen = binary.Uvarint(bj.Value[valOff+1:])
```

Comment:
`binary.Uvarint` should have handled cases of `strLen > utf8.RuneSelf`, why manual? For performance?

Reply:
I think it's fine to use the `binary.Uvarint` directly. I also modified the existing `GetString` implementation to use `binary.Uvarint` without the special condition.

PTAL